pallets/loans/rpc/runtime-api/src/lib.rs

```diff
@@ -15,12 +15,13 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 use codec::Codec;
-use primitives::{Liquidity, Shortfall};
-use sp_runtime::DispatchError;
+use primitives::{CurrencyId, Liquidity, Rate, Ratio, Shortfall};
+use sp_runtime::{DispatchError, FixedU128};
 
 sp_api::decl_runtime_apis! {
     pub trait LoansApi<AccountId> where
         AccountId: Codec, {
         fn get_account_liquidity(account: AccountId) -> Result<(Liquidity, Shortfall), DispatchError>;
+        fn get_market_status(asset_id: CurrencyId) -> Result<(Rate, Rate, Rate, Ratio, u128, u128, FixedU128), DispatchError>;
```

Comment:
u128 may not be safe as rpc return

Reply:
we have same issue in amm router I guess, we can check getBestRoute rpc

Reply:
👌🏻